.gitignore

```diff
@@ -1,22 +1,16 @@
 .DS_Store
 .vscode/
+.vs/
 *.code-workspace
-obj
+obj/
 basic/basic
-standard/obj
```

Comment:
why did you remove this from .gitignore?

Reply:
`obj/` replaces all removed folder specific `whatever/obj` rules